src/ProjectNeon/Assets/Data/Enemies/AI/DumbAI.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DumbAI : TurnAI
+{
+    /**
+     * The AI member
+     */
+    private Member me;
```

Comment:
How is this initialized? It's not possible in the editor, nor in code, presently.